velox/exec/GroupingSet.cpp

```diff
@@ -96,6 +96,26 @@ GroupingSet::~GroupingSet() {
   }
 }
 
+std::unique_ptr<GroupingSet> GroupingSet::createForMarkDistinct(
+    std::vector<std::unique_ptr<VectorHasher>>&& hashers,
+    OperatorCtx* FOLLY_NONNULL operatorCtx) {
+  tsan_atomic<bool> nonReclaimableSection{false};
+  return std::make_unique<GroupingSet>(
+      std::move(hashers),
+      /*preGroupedKeys*/ std::vector<column_index_t>{},
+      /*aggregates*/ std::vector<std::unique_ptr<Aggregate>>{},
+      /*aggrMaskChannels*/ std::vector<std::optional<column_index_t>>{},
+      /*channelLists*/ std::vector<std::vector<column_index_t>>{},
+      /*constantLists*/ std::vector<std::vector<VectorPtr>>{},
+      /*intermediateTypes*/ std::vector<TypePtr>{},
+      /*ignoreNullKeys*/ true,
```

Comment:
I noticed that tests didn't test inputs with nulls. I added these tests and they failed. I changed this boolean to 'false' to fix. Any particular reason this flag was set to true?